tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/AspNet/OwinWebApi2Tests.cs

```diff
@@ -126,6 +126,7 @@ public OwinFixture()
             {
                 _httpClient = new HttpClient();
                 _httpClient.DefaultRequestHeaders.Add(HttpHeaderNames.TracingEnabled, "false");
+                _httpClient.DefaultRequestHeaders.Add(HttpHeaderNames.UserAgent, "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36");
```

Comment:
Out of interest, did you try running these tests without your fix first, to confirm the faulty behaviour?

Reply:
Yes. I added the tests and fixed it afterwards. And indeed, it splits at each space (approximatively)